### PR TITLE
[SLC-2025] Open CFP and add dates

### PR DESCRIPTION
**Pull request by Clait**
- Open CFP/dates
- Add conference dates
- Add propose page nav element
- Update logo 
- Remove Alex as an organizer for 2025

### CODE REVIEW

File: data/events/2025/salt-lake-city/main.yml [hunk 1]
```diff
@@ -73,9 +69,9 @@ team_members: # Name is the only required field for team members.
       employer: "Intermountain Healthcare"
       linkedin: "https://www.linkedin.com/in/claiton-weeks/"
       twitter: "vsc_sre"
-    - name: "Alex Bilimoria"
-      employer:  "Quotient Technology"             # "Senior Software Engineer"
-      linkedin: "https://www.linkedin.com/in/alex-c-bilimoria/"
+    # - name: "Alex Bilimoria"
+    #   employer:  "Quotient Technology"             # "Senior Software Engineer"
+    #   linkedin: "https://www.linkedin.com/in/alex-c-bilimoria/"
```
Comment (toshywoshy) [inline data/events/2025/salt-lake-city/main.yml:74]: This requires an email to info

**Review by toshywoshy — CHANGES_REQUESTED** [review]

File: data/events/2025/salt-lake-city/main.yml (hunk 1 above)
Comment (Clait) [reply data/events/2025/salt-lake-city/main.yml:74]: > This requires an email to info

Sent!  Thanks.

**Comment by phrawzty** [comment]
📧 received and processed

File: data/events/2025/salt-lake-city/main.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: "2025-salt-lake-city" # The name of the event. Four digit year with the ci
 year: "2025" # The year of the event. Make sure it is in quotes.
 city: "Salt Lake City" # The displayed city name of the event. Capitalize it.
 event_twitter: "devopsdaysslc" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
-event_logo: "logo-square.jpg"
+event_logo: "logo2.png"
```
Comment (phrawzty) [inline data/events/2025/salt-lake-city/main.yml:5]: There is no file named `logo2.png` in this pull request, nor anywhere in the current repository. Please either include this file, or adjust the filename to one that exists.

**Review by phrawzty — CHANGES_REQUESTED** [review]

**Review by toshywoshy — APPROVED** [review]

**Pull request merged**